### ISSUE TITLE
`react/jsx-wrap-multilines` should throw when using arrow functions

### ISSUE THREAD

**Issue by eTorAken** [yannickcr/eslint-plugin-react#874]
Hi,

When activating `react/jsx-wrap-multilines` (with `declaration`, `assignment` and `return` set to `true`), I'm a bit surprised that this code does not throw any warning:
```js
const MyComponent= () =>
  <AChildComponent className={style.aChildComponent}>
    <AnotherChildComponent />
  </AChildComponent>;
```

Imho, it should be written that way:
```js
const MyComponent= () => (
  <AChildComponent className={style.aChildComponent}>
    <AnotherChildComponent />
  </AChildComponent>
);
```

**Comment by SpainTrain** [comment]
Think this is dupe of #790?

**Comment by eTorAken** [comment]
Indeed, thank you for noticing it @SpainTrain, I close this one.
